EDITORS.md

```diff
@@ -142,29 +142,7 @@ require("lspconfig").ruby_lsp.setup({
 
 ## LazyVim LSP
 
-For LazyVim, you can add the ruby-lsp by creating a file in your plugins folder (`~/.config/nvim/lua/plugins/ruby_lsp.lua`) and adding the following:
-
-```lua
--- ~/.config/nvim/lua/plugins/ruby_lsp.lua
-
-return {
-  {
-    "neovim/nvim-lspconfig",
-    ---@class PluginLspOpts
-    opts = {
-      ---@type lspconfig.options
-      servers = {
-        -- disable solargraph from auto running when you open ruby files
-        solargraph = {
-          autostart = false
-        },
-        -- ruby_lsp will be automatically installed with mason and loaded with lspconfig
-        ruby_lsp = {},
-      },
-    },
-  },
-}
-```
+[As of v12.33.0](https://github.com/LazyVim/LazyVim/pull/3652), Ruby LSP is the default LSP for Ruby and no configuration should be needed.
```

Comment:
Does it make sense to keep the previous instructions for people on older versions? Or is it common practice to be on the latest one for LazyVim?

Reply:
It seems like [an easy update](https://github.com/LazyVim/LazyVim/issues/1563#issuecomment-1744228821):

> `LazyVim` itself is the same as a regular plugin, so there is no need for a special update command. You update LAzyVim like you update any other plugin using `:Lazy`, the plugin mamager.

Reply:
Let's see if people object to upgrading, if so we can re-add the old information.